packages/allure-codeceptjs/src/index.ts

```diff
@@ -156,8 +158,6 @@ class AllureReporter {
   }
 
   testStarted(test: CodeceptTest & { tags: string[] }) {
-    this.currentTest = test;
```

Comment:
Maybe we should keep it? Just in case something went wrong like no `suite.ctx.currentTest` is set?

Reply:
@baev agree,
rolled back this.currentTest = test;